### PR TITLE
fix(i18n): translation key

### PR DESCRIPTION
**Pull request by AugustinMauroy**
fix translation key to feat with this schema: `{folder}.{subFolder}.{componentFolder}.{partOfComponent}`

# Related issues

no related issue

### CODE REVIEW

File: i18n/locales/en.json [hunk 1]
```diff
@@ -43,19 +43,19 @@
...
   "components.common.languageSelector.button.title": "Switch Language",
   "components.blog.blogCard.author.by": "by",
-  "components.footer.links.trademark": "Trademark Policy",
-  "components.footer.links.privacy": "Privacy Policy",
-  "components.footer.links.codeOfConduct": "Code of Conduct",
-  "components.footer.links.security": "Security",
-  "components.footer.links.about": "About",
-  "components.footer.links.blog": "Blog",
-  "components.footer.links.github": "GitHub",
+  "components.section.newfooter.trademark": "Trademark Policy",
```
Comment (ovflowd) [inline i18n/locales/en.json]: Name of the subfolder is `sections` not `section` 👀

File: i18n/locales/en.json [hunk 2]
```diff
@@ -43,19 +43,19 @@
...
+  "components.section.newfooter.privacy": "Privacy Policy",
+  "components.section.newfooter.codeOfConduct": "Code of Conduct",
+  "components.section.newfooter.security": "Security",
+  "components.section.newfooter.about": "About",
+  "components.section.newfooter.blog": "Blog",
+  "components.section.newfooter.github": "GitHub",
   "components.prevnextlink.next": "NEXT",
```
Comment (ovflowd) [inline i18n/locales/en.json]: Could you also fix this one? I know it's unrelated, but I'd appreciate!

File: components/Sections/NewFooter/index.tsx [hunk 3]
```diff
@@ -35,22 +35,22 @@ const Footer = () => (
...
       <li>
         <LocalizedLink className={styles.link} href="/about/security">
-          <FormattedMessage id="components.footer.links.security" />
+          <FormattedMessage id="components.section.newFooter.security" />
         </LocalizedLink>
       </li>
       <li>
         <LocalizedLink className={styles.link} href="/about">
-          <FormattedMessage id="components.footer.links.about" />
+          <FormattedMessage id="components.section.newFooter.about" />
```
Comment (ovflowd) [inline components/Sections/NewFooter/index.tsx]: You forgot to update the usage of `section` to `sections`. Again, be extra attentive when submitting changes...

File: i18n/locales/en.json [hunk 4]
```diff
@@ -43,19 +43,19 @@
...
-  "components.footer.links.github": "GitHub",
-  "components.prevnextlink.next": "NEXT",
-  "components.prevnextlink.previous": "PREV",
-  "pages.index.features.javascript.title": "JavaScript",
-  "pages.index.features.javascript.description": "Node.js provides support for the JavaScript programming language",
-  "pages.index.features.openSource.title": "Open Source",
-  "pages.index.features.openSource.description": "Node.js is open source and actively maintained by contributors all over the world",
-  "pages.index.features.everywhere.title": "Everywhere",
-  "pages.index.features.everywhere.description": "Node.js has been adapted to work in a wide variety of places"
+  "components.section.newFooter.trademark": "Trademark Policy",
```
Comment (shanpriyan) [inline i18n/locales/en.json]:
```suggestion
  "components.sections.newFooter.trademark": "Trademark Policy",
```

Also update the key as `sections` in `NewFooter` component too (as ovflowd mentioned)

**Review by shanpriyan — CHANGES_REQUESTED** [review]

**Review by shanpriyan — APPROVED** [review]
LGTM!

**Review by ovflowd — APPROVED** [review]

**Pull request merged**